### PR TITLE
Cleanup CI workflows, Remove Twenty CLI, Add Danger.js

### PR DESCRIPTION
**Pull request by FelixMalfait**
- Cleanup CI workflows
- Remove Twenty CLI
- Add Danger.js

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +0,0 @@
-server/node_modules/
```
Comment (charlesBochet) [inline .dockerignore:1]: @FelixMalfait are you sure about removing this folder? Have you tried to build the docker and checked the image size?

Comment (charlesBochet) [reply .dockerignore:1]: I see that you moved it to server folder. I don't think it works:
"When you run a build command, the build client looks for a file named .dockerignore in the root directory of the context"

Comment (FelixMalfait) [reply .dockerignore]: You are right! I had not tested it yet

Comment (FelixMalfait) [reply .dockerignore]: Thanks!

**Pull request merged**